include/nccl_ofi_rdma.h

```diff
@@ -155,11 +158,23 @@ static inline size_t nccl_net_ofi_rdma_ctrl_msg_size(size_t num_rails, bool use_
 	return offsetof(nccl_net_ofi_rdma_ctrl_msg_t, short_buff_mr_key) + num_rails * rkey_len;
 }
 
+/* Message from receiver to sender indicating sender can close resources */
+typedef struct nccl_net_ofi_rdma_close_msg {
+	/* Message type, must be NCCL_OFI_RDMA_MSG_CLOSE */
+	uint16_t type:NCCL_OFI_RDMA_CTRL_TYPE_BITS;
+	uint16_t pad:(16 - NCCL_OFI_RDMA_CTRL_TYPE_BITS);
```

Comment:
sizeof (uint16_t) - NCCL_OF_RDMA_CTRL_TYPE_BITS, although you don't actually need to add the padding explicitly.

Reply:
sizeof(uint16_t) is two, you mean 8*sizeof(uint16_t)

Reply:
sizeof(uint16_t) can also legally be `unsigned int`, so is not even necessarily two. There exist cursed platforms where `static_assert(sizeof(uint16_t) == 1)`. :-)